server.js

```diff
@@ -4222,6 +4223,43 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// ImageLayers.io integration.
+camp.route(/^\/imagelayers\/([^\/]+)\/([^\/]+)\/([^\/]*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1];
+  var repo = match[2];
+  var tag = match[3]
+  var format = match[4];
+  if (user === '_') {
+    user = 'library';
+  }
+  var path = user + '/' + repo;
+  var badgeData = getBadgeData('imagelayers', data);
+  var options = {
+    method: 'POST',
+    json: {
+      "repos": [{"name": path, "tag": tag}]
+    },
```

Comment:
I believe, from [request's documentation](https://www.npmjs.com/package/request), that the `json` entry is meant to either be falsy or true.